Automattic/wp-calypso
# Site editor: Link "Manage all pages" to preferred view

Fixes https://github.com/Automattic/wp-calypso/issues/81390

## Proposed Changes

Ensures that the Site Editor's "Manage all pages" link leads to the preferred view.

<img width="1345" alt="Screenshot 2023-09-07 at 14 42 10" src="https://github.com/Automattic/wp-calypso/assets/1233880/9ee00fbc-1535-4fb5-b097-ee400a3421f3">

## Testing Instructions

- Apply these changes to your sandbox: `install-plugin.sh wpcom-block-editor update/site-editor-manage-all-pages`.
- Sandbox `widgets.wp.com` and a test site.
- Open the site editor of the sandboxed test site.
- Select "Pages" and click on the "Manage all pages" link.
- Make sure you land in the preferred view of the "Pages > All pages" screen (it's Calypso by default, but you might have chosen a different one).
- While in the "Pages > All pages" screen, use the "View" toggle to switch to the opposite view.
- Open the site editor and use the "Manage all pages" link once again.
- Make sure you land in your new preferred view.
